apps/fishing-map/project.json

```diff
@@ -14,20 +14,10 @@
     "deploy": {
       "executor": "nx:run-commands",
       "options": {
-        "commands": ["nx build fishing-map --parallel", "nx docker-prepare fishing-map"],
+        "commands": ["nx build fishing-map --parallel"],
         "parallel": false
       }
     },
-    "docker-prepare": {
-      "executor": "nx:run-commands",
-      "outputs": [],
-      "options": {
-        "commands": [
-          "cp config/entrypoint.sh dist/apps/fishing-map",
-          "cp apps/fishing-map/nginx.conf dist/apps/fishing-map"
-        ]
-      }
-    },
     "compile": {
```

Comment:
I think we can update this as the export process is not needed anymore.
I got this warning while testing locally:
```
warn  - Statically exporting a Next.js application via `next export` disables API routes.
This command is meant for static-only hosts, and is not necessary to make your application static.
``` 
So I opened [this PR](https://github.com/GlobalFishingWatch/frontend/pull/1825) to test the deploy and seems to be working as expected but [the build runs](https://console.cloud.google.com/cloud-build/builds;region=global/0fb40ac1-e704-4a23-a317-31ebc39f6f17?project=world-fishing-827) faster (see 3 mins Vs 5) so could you please take a look? and I'll migrate the rest of the apps if you agree

Reply:
Thanks! LGTM, go ahead and migrate the rest of the apps 💯